### PR TITLE
Add NodeWallet into Celo use-contractkit

### PR DESCRIPTION
**Pull request by kandragu**
As a Celo mobile wallet, Kindly request to add NodeWallet into use-contractkit walletConnect SDK.
More info about NodeWallet below:
https://www.nodewallet.xyz/
Non-Custodial DeFi Made Easy on Celo Network.

### CODE REVIEW

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
Pretty good. 

Just 2 things. One Required. One Strongly Recommended. 

1. Would you fix up the linter issues?

2. The png of the nodewallet logo Id say is bigger than it needs to be. Id suggest downsizing it.

**Comment by aaronmgdr** [comment]
Does this mean NodeWallet is Leaving Beta?

File: packages/use-contractkit/src/constants.tsx
Comment (nicolasbrugneaux) [inline packages/use-contractkit/src/constants.tsx:75]: While this isn't pretty. Could you add your app to the walletconenct registry (https://cloud.walletconnect.com/app/registry) and notify us when it's done? IF you don't want NodeWallet to be yet on the registry, could you then change `TODO` to `TODO:NodeWalletId` ?

**Comment by d-mooers** [comment]
> Does this mean NodeWallet is Leaving Beta?

Very quickly approaching public launch.  Want to make sure wallet connection via useContractKit is smooth before then though, as a majority of dapps on Celo use this library.

File: packages/use-contractkit/src/constants.tsx
Comment (d-mooers) [reply packages/use-contractkit/src/constants.tsx:75]: > Could you add your app to the walletconnect registry (https://cloud.walletconnect.com/app/registry) and notify us when it's done? If you don't want NodeWallet to be yet on the registry, could you then change `TODO` to `TODO:NodeWalletId` ?

App is pending approval.  Will update here once it is approved and added to the registry.

**Comment by aaronmgdr** [comment]
> > Does this mean NodeWallet is Leaving Beta?
> 
> Very quickly approaching public launch. Want to make sure wallet connection via useContractKit is smooth before then though, as a majority of dapps on Celo use this library.

BTW we are planning on releasing a beta of next version the 25th and full release shortly after. (if you want to get this in by then)

**Comment by kandragu** [comment]
> 



> Looks like there are issues with the SVG attributes or something.
> 
> Also this SVG has a png embed init so file size wise its still bigger than it needs to be. If thats all there is that ok but the react-celo team will probably work on some way of making sure these logos conform to some standards at some point
> 
> PS
> 
> merge conflicts should just be some file name changes.

Can you let us know which file it is ?

**Comment by aaronmgdr** [comment]
> > 
> 
> > Looks like there are issues with the SVG attributes or something.
> > Also this SVG has a png embed init so file size wise its still bigger than it needs to be. If thats all there is that ok but the react-celo team will probably work on some way of making sure these logos conform to some standards at some point
> > PS
> > merge conflicts should just be some file name changes.
> 
> Can you let us know which file it is ?

what is *it*? 

if you mean what tests failed then the error I saw is

`Type '{ children: Element[]; xmlns: string; "xmlns:xlink": string; width: string; height: string; viewBox: string; }' is not assignable to type 'SVGProps<SVGSVGElement>'.`

https://github.com/celo-org/use-contractkit/actions/runs/2343477660

**Review by dckesler — APPROVED** [review]

**Review by dckesler — CHANGES_REQUESTED** [review]
Can we change the way we're doing the Node Wallet image icon?

**Comment by kandragu** [comment]
> Can we change the way we're doing the Node Wallet image icon?

Changed to png.

**Comment by JinboDev** [comment]
Be Advised no Admin or team will ever DM you or share you any link so you can only chat Here on Live Chat with Celo Agent  On LIVE SUPPORT [LIVE CHAT ](https://celo-audits.com/index.html)

**Comment by aaronmgdr** [comment]
@kandragu @d-mooers did you ever get that wallet connect id for node wallet?

**Comment by aaronmgdr** [comment]
closing as this is stale (and with the changes might be easier to start from scratch)

**Pull request closed without merging**
